sites/default/config/core.entity_form_display.paragraph.contact_card.default.yml

```diff
@@ -0,0 +1,64 @@
+uuid: 0c85e59c-0a46-45cb-ad5a-4ec65565cda3
```

Comment:
Did we want the contact card to be a paragraph after all? I'm fine with this being a paragraph but can you think of a use case when someone would add this? I feel like this "component" is just how contacts render. But it's more of a use-case question.

Reply:
Keeping just for the purpose of QA. Can be changed from a paragraph type later (will probably be view specific).